src/base.config.ts

```diff
@@ -44,6 +45,12 @@ export const packageName = packageJson.name || '';
 
 const tsLintPath = path.join(basePath, 'tslint.json');
 const tsLint = existsSync(tsLintPath) ? require(tsLintPath) : false;
+const tsLintExcludes =
```

Comment:
I find this a little hard to parse, perhaps we can make it clearer using an if block?

```tsx
let tsLintExcludeFiles = [];
if (tsLint && tsLint.linterOptions && tsLint.linterOptions.excludes) {
    const { linterOptions: { exclude } } = tsLint;
    tsLintExcludeFiles = globby.sync(exclude).map((file) => path.resolve(file)));
}
```